lib/g10_certification_test_kit/bulk_data_group_export.rb

```diff
@@ -0,0 +1,258 @@
+
+        skip "Server took more than #{timeout} seconds to process the request." if response[:status] == 202
+
+        assert response[:status] == 200, "Bad response code: expected 200, 202, but found #{response[:status]}."
+        assert request.response_header('content-type')&.value&.include?('application/json'),
+               'Content-Type not application/json'
+
+        response_body = JSON.parse(response[:body])
+
+        ['transactionTime', 'request', 'requiresAccessToken', 'output', 'error'].each do |key|
+          assert response_body.key?(key), "Complete Status response did not contain \"#{key}\" as required"
+        end
+
+        output status_response: response[:body]
+      end
+    end
+
+    test do
+      title 'Bulk Data Server returns output with type and url for status complete'
+      description <<~DESCRIPTION
+        The value of output field is an array of file items with one entry for each generated file.
+        If no resources are returned from the kick-off request, the server SHOULD return an empty array.
+
+        Each file item SHALL contain the following fields:
+
+        * type - the FHIR resource type that is contained in the file.
+
+        Each file SHALL contain resources of only one type, but a server MAY create more than one file for each resource type returned.
+
+        * url - the path to the file. The format of the file SHOULD reflect that requested in the _outputFormat parameter of the initial kick-off request.
+      DESCRIPTION
+      # link 'http://hl7.org/fhir/uv/bulkdata/export/index.html#response---complete-status'
+
+      input :status_response
+
+      output :status_output
+
+      run do
+        assert status_response.present?, 'Bulk Data Server status response not found'
+
+        status_output = JSON.parse(status_response)['output']
+        assert status_output, 'Bulk Data Server status response does not contain output'
+
+        output status_output: status_output.to_json
+
+        status_output.each do |file|
+          ['type', 'url'].each do |key|
+            assert file.key?(key), "Output file did not contain \"#{key}\" as required"
+          end
+        end
+      end
+    end
+
+    test do
+      title 'Bulk Data Server returns requiresAccessToken with value true'
+      description <<~DESCRIPTION
+        Bulk Data Server SHALL restrict bulk data file access with access token
+      DESCRIPTION
+      # link 'http://hl7.org/fhir/uv/bulkdata/export/index.html#response---complete-status'
+
+      input :status_response
+
+      run do
+        assert status_response.present?, 'Bulk Data Server status response not found'
+
+        requires_access_token = JSON.parse(status_response)['requiresAccessToken'].to_s.downcase
+        output requires_access_token: requires_access_token
+
+        assert requires_access_token.present? && requires_access_token == 'true',
+               'Bulk Data file server access SHALL require access token'
+      end
+    end
```

Comment:
```suggestion
```

Reply:
Deleted this test.

Reply:
Actually deleted this too hastily. It ended up breaking quite a few things, and I thought there would be a quick and easy fix, but now I think our best bet is to just get this PR in with this test and then figure out removing requiresAccessToken after.